include/scippp/model.hpp

```diff
@@ -250,6 +251,26 @@ class Model {
         }
     }
 
+    /**
+     * Writes original problem to file.
+     *
+     * @since 1.1.0
+     * @param filename output file name including extension
+     * @param genericNames using generic variable and constraint names?
```

Comment:
Could you document a little more clearly what "generic names" means?

Reply:
docu enhanced and test added